_layouts/default.html

```diff
@@ -63,6 +63,54 @@
         <script defer src="/js/bootstrap.min.js"></script>
         <script defer src="/js/scripts.js"></script>
 
+        <!-- Schema.org JSON-LD Markup for Search Engine Optimization -->
+        <script type='application/ld+json'>
+        {
+            "@context": "https://schema.org/",
+            "@type": "SoftwareApplication",
+            "applicationCategory": "FinanceApplication",
+            "name": "Bisq",
+            "image": "https://bisq.network/images/bisq-logo.svg",
+            "availableOnDevice": "Desktop",
+            "author": {
+            "@type": "Project",
+            "url": "https://bisq.network/dao/",
```

Comment:
```suggestion
            "url": "https://bisq.network/",
```

See reasoning in comment below about referring to the author as Bisq instead of the Bisq DAO.